statusboard/src/components/Map/Map.js

```diff
@@ -124,6 +140,9 @@ export default function Map({ brigadeData, filterOpts, setFilterOpts }) {
         onClick={() => {
           setZoom(defaultZoom);
           setCenter(defaultCenter);
+          localStorage.setItem('zoom', defaultZoom);
+          localStorage.setItem('lat', defaultCenter[0]);
+          localStorage.setItem('lng', defaultCenter[1]);
```

Comment:
nitpick: You might want to use a pattern similar to https://gustavostraube.wordpress.com/2019/05/29/custom-setters-with-react-usestate-hook/ here and see the `localStorage.setItem` as a side-effect of `setZoom` and `setCenter`.
We'd probably have to wrap it in a `useMemo` or `useCallback` since we are dealing with JSX (I suspect ESLint will complain about potential performance issues otherwise)

If we add a "center on my location" in the future it would reduce the risk of forgetting to update the `localStorage` and the code would be more DRY.

Reply:
This is very cool, I learned something today :)